### ISSUE TITLE
Having Trouble with Downloading

### ISSUE THREAD

**Issue by nealadam17** [mattpannella/pocket-updater-utility#92]
I successfully downloaded the pocket_updater and ran it, initially I ran individual cores and that was successful, but then I realized I would like to try just downloading them all. There were a lot so I tried deleting the cores on my SD card and choose to download all when I brought back up the updater. When I did this it simply said it was done and would bring me back to the main menu. I noticed I still had files in assets so I deleted those and tried again, with the same result. I tried deleting everything off the SD card and redownload the updater. When it asks me if I want to automatically install cores and I choose Y, it just brings me to the main menu. Choosing cores doesn't seem to do anything, and when I chose 0 for updating everything, it actually began the process again but there are a lot of errors. To many to go over but, for example, some say, "Error communicating with Github API" and others say, "uh oh something went wrong" 

The weird thing is a handful of cores seem to have gone through. I've got assets for arduboy, asteroids, nes, pce, poke_mini, and snes. Which, actually, are ones I chose not to download on one attempt when doing the manual asking about each core method. 

Sorry if this is all obvious, I just don't know what to do.

**Comment by nealadam17** [comment]
I don't even remember everything I've tried at this point, but I do remember that initially the 1st time I actually did successfully get the majority of the arcade cores, and I saw the GB, GBC, and GBA bios files inside of Assets the readme file said needed to be there before using. Originally when I first tried to delete things and redownload the whole package, nothing would happen, and if I put my SD card back in the anologue pocket, it would still show everything in the FPGA on the system even though I thought I deleted it. However, when I put the card back in the pc I saw that those bios folders were deleted and trying to use the updater never seemed to bring them back, that's why I ended up deleting absolutely everything off the card and starting from scratch, which brought me to the level of success I mentioned in the first post. If I try to select just GB GBC and GBA for updating cores nothing happens.

**Comment by nealadam17** [comment]
I made one more attempt by deleting everything on the card and using the most recent version of updater, when asked chose A, chose what individual cores I want put on the card, and when I hit enter it simply brought me back to "Welcome to Flavor Town"

**Comment by nealadam17** [comment]
Ok well, I decided to give it one more go, I deleted everything from the sd card, I deleted the updater from there and my computer and recycle bin, and when I redownloaded the updater and chose to update everything... it did. Everything is there on the card now. I have no idea what the problem was before. Sorry for bothering you with this.

While I have you here, when I find ROMS online, where should I put them on the card? Should I make a folder titled ROM? Or should all individually be put on the root like I did with the pocket_updater?
